src/charm.py

```diff
@@ -326,7 +355,6 @@ def _pull_licenses(self, container: Container) -> None:
         """Pull licenses from workload."""
         licenses = [
             "snap",
-            "rock",
```

Comment:
Minor question - looks like this change is not related to the PR topic. Is this correct?

Reply:
ooh this was a mistake - thank you for noticing 🙏